README.md

```diff
@@ -162,6 +162,84 @@ $sumThree(7, 42, 666);
 
 All invocations will yield the same result.
 
+#### Avoid billion dollar mistake by using `Option`
```

Comment:
Not every body knows, that `null` reference is called "billion dollar mistake". Maybe change this to:

```
Avoid billion dollar mistake (`null`) by using `Option`
```

Reply:
Please use `###` level of headline and add it into TOC.